### PR TITLE
Add -timeout flag for customizable HTTP Request timeouts

### PR DESCRIPTION
**Pull request by lc**
Hi,

Like the title implies, this PR adds the command line flag `-timeout` which allows users to add a custom HTTP request timeout. I've had several use-cases for it recently so I figured I would add this.

By default, it is set to `10` seconds.

### CODE REVIEW

**Comment by joohoi** [comment]
Thanks for the PR! It's definitely a good feature to have, we discussed it recently in #4 with @SolomonSklash as well.

**Comment by lc** [comment]
Added!

File: main.go [hunk 1]
```diff
@@ -78,6 +78,7 @@ func main() {
 	flag.BoolVar(&conf.FollowRedirects, "r", false, "Follow redirects")
 	flag.BoolVar(&conf.AutoCalibration, "ac", false, "Automatically calibrate filtering options")
 	flag.IntVar(&conf.Threads, "t", 40, "Number of concurrent threads.")
+	flag.IntVar(&conf.Timeout, "timeout", 10, "HTTP request timeout. Default: 10")
```
Comment (joohoi) [inline main.go]: The default is printed automatically in the CLI help so the `Default: 10` part can be removed.

Comment (joohoi) [reply main.go]: Oh, and we could mention that the unit is second, so something along the lines of
```
HTTP request timeout in seconds.
```

Though this is probably obvious for 99% of the users, it doesn't hurt to be more explicit.

Comment (lc) [reply main.go]: Done

**Comment by lc** [comment]
Should be all good to go now

**Pull request merged**